android/src/main/java/com/actiotech/twiliovideorn/TwilioVideoModule.java

```diff
@@ -182,6 +184,17 @@
     private final Map<String, Pair<RemoteDataTrack, RemoteParticipant>> remoteDataTrackMap =
             new HashMap<>();
 
+    private final TimerTask requestStatsTask = new TimerTask() {
+        @Override
+        public void run() {
+            if (room != null) {
+                room.getStats(TwilioVideoModule.this);
```

Comment:
#nostalgia
I miss optional chaining from Swift:

`room?.getStats(...);`

It also would work in a TypeScript, but it still would complaint that this should be an expression and must not be called.